starknet_py/net/signer/keypair_signer.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+from typing import List
+
+from starkware.crypto.signature.signature import (
+    private_to_stark_key,
+)
+from starkware.starknet.core.os.transaction_hash.transaction_hash import (
+    calculate_transaction_hash_common,
+)
+
+from starknet_py.net.models import AddressRepresentation, StarknetChainId, Transaction
+from starknet_py.net.signer.base_signer import BaseSigner
+from starknet_py.utils.crypto.facade import message_signature
+
+
+@dataclass
+class KeyPair:
+    private_key: int
+    public_key: int
+
+    @staticmethod
+    def from_private_key(key: int) -> "KeyPair":
+        return KeyPair(private_key=key, public_key=private_to_stark_key(key))
+
+
+class KeypairSigner(BaseSigner):
```

Comment:
There's a lot of algorithms that involve a key pair. What if we named this DefaultSigner?

Reply:
I've considered that but I thought `DefaultSigner` isn't descriptive enough - can change it to that though if you think it would be better that way.

Reply:
Then name it after signature method used in starknet

Reply:
How about something like `EllipticCurveSigner`?

Reply:
Or `StarkCurveSigner`?